File: examples-sdk/csharp/002-basic-vilapp-gateway/Main.cs
// 002-basic-vilapp-gateway — C# SDK equivalent
// Compile: vil compile --from csharp --input 002-basic-vilapp-gateway/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("vil-app-gateway", 3081);
var gw = new ServiceProcess("gw");
gw.Endpoint("POST", "/trigger", "trigger_handler");
server.Service(gw);
server.Compile();

File: examples-sdk/swift/002-basic-vilapp-gateway/main.swift
// 002-basic-vilapp-gateway — Swift SDK equivalent
// Compile: vil compile --from swift --input 002-basic-vilapp-gateway/main.swift --release

let server = VilServer(name: "vil-app-gateway", port: 3081)
let gw = ServiceProcess(name: "gw")
gw.endpoint(method: "POST", path: "/trigger", handler: "trigger_handler")
server.service(gw)
server.compile()
